docs/preprocess.rst

```diff
@@ -125,7 +135,44 @@ This pipeline can be run through the functions saved in ``site_pipeline``. Each
 entry in "process_pipe" key will be used to generate a Preprocess module based
 on the name it is registered to. These entries will then be run in order through
 the processing pipe. The ``process`` function is always run before the
-``calc_and_save`` function for each module.
+``calc_and_save`` function for each module. The ``plot`` function can be run after
+``calc_and_save`` when ``plot: True`` for a module that supports it. Current modules
+with plotting: ``DetBiasFlags``, ``EstimateHWPSS``, ``SSOFootprint``.
+
+Example TOD Pipeline Configuration File
```

Comment:
Maybe you should call this "Example Obs Pipeline Configuration File" in contrast to the "TOD Pipeline Configuration File" above. Now it looks like two entries with the same name in the tree
<img width="311" alt="Screenshot 2024-03-13 at 6 22 26 PM" src="https://github.com/simonsobs/sotodlib/assets/25913911/e47ffda2-629c-4211-8072-3efcf94af796">